applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -813,6 +812,26 @@ static void handle_config_channel_peers_req(const struct config_event *event)
 		break;
 	}
 
+	case CONFIG_STATUS_GET_PEERS_CACHE:
+	{
+		BUILD_ASSERT(ARRAY_SIZE(peripherals) <= CONFIG_CHANNEL_FETCHED_DATA_MAX_SIZE);
+		BUILD_ASSERT(ARRAY_SIZE(peripherals) == ARRAY_SIZE(peripheral_cache));
+
+		size_t pos = 0;
+		struct config_event *rsp = generate_response(event,
+							     CONFIG_CHANNEL_FETCHED_DATA_MAX_SIZE);
+
+		memcpy(&rsp->dyndata.data[pos], peripheral_cache, sizeof(peripheral_cache));
+		pos += sizeof(peripheral_cache);
+
+		memset(&rsp->dyndata.data[pos], 0x00, CONFIG_CHANNEL_FETCHED_DATA_MAX_SIZE - pos);
+		pos = CONFIG_CHANNEL_FETCHED_DATA_MAX_SIZE;
```

Comment:
nit: i think this line is unnecessary as `pos` is not further used here.

Reply:
To be honest I preferred to update it anyway (it does not cost much and data consistency is ensured :))